sql/src/test/java/io/crate/analyze/where/EqualityExtractorTest.java

```diff
@@ -333,4 +346,62 @@ public void testPK2_Eq1() throws Exception {
         assertNull(matches);
     }
 
+    @Test
+    public void testPK1_AnyEq() throws Exception {
```

Comment:
hm not so meaningful method names, more verbose wouldn't hurt ;) also all other tests in this class